lib/stat_tracker.rb

```diff
@@ -1,16 +1,20 @@
 require 'csv'
 class StatTracker
   attr_reader :game_teams,
-              :games,
+              :game_rows,
               :teams
 
   def initialize(locations)
-    game_teams = CSV.read locations[:game_teams], headers: true, header_converters: :symbol
```

Comment:
I wouldn't remove these locations since we'll need access to these in the other classes.